content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
 
 normal bounded
+: _not available in the "RGB (Scene)" color space_
 : This blend mode is the same as “normal”, except that the input and output data are clamped to a particular min/max value range. Out-of-range values are effectively blocked and are not passed to subsequent modules. Sometimes this helps to prevent artifacts. However, in most cases (e.g. highly color-saturated extreme highlights) it is better to let unbound values travel through the pixelpipe to be properly handled later. The “normal” blend mode is therefore usually preferred.
 
-lightness
-: This blend mode mixes lightness from the input and output images. Color (chroma and hue) is taken unaltered from the input image.
+# arithmetic modes
 
-chroma
-: This blend mode mixes chroma (saturation) from the input and output images. Lightness and hue are taken unaltered from the input image.
+addition
+: Add together the pixel values of the input and output images, lightening the output. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum".
 
-hue
-: This blend mode mixes hue (color tint) from the input and output images. Lightness and chroma are taken unaltered from the input image.
+subtract
+: Subtract the pixel value of the _output_ from the _input_. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum". Pixel values less than 0 are set to 0.
 
-color
-: This blend mode mixes color (chroma and hue) from the input and output images. Lightness is taken unaltered from the input image. 
+subtract reverse
+: _only available in the "RGB (Scene)" color space_
+: Subtract the pixel value of the _input_ from the _output_. In this case, the "blend fulcrum" parameter of the "RGB (scene)" color space is applied to the input image.
 
-: _Caution: When modules drastically modify hue (e.g. when generating complementary colors) this blend mode can result in strong color noise._
+multiply
+: Multiply the pixel values of the input and output together. As all pixel values are between 0 and 1.0, the resulting output will always be darker. When blending in the "RGB (scene)" color space, this value is further multiplied by a value proportional to the "blend fulcrum".
+
+multiply reverse
+: _only available in the "RGB (Scene)" color space_
+: This mode works in the same way as the "multiply" mode except that in the `final_output` equation given at the top of this page, the `input` parameter is replaced with `module_output` (the output of the module before blending is applied).
+
+divide
+: Divide the pixel values of the input by the output. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum".
```

Comment:
You may add "Since this is the inverse of the multiply mode, it will darken where multiply brightens, and the other way around. Everything else works essentially the same.".